### ISSUE TITLE
v3.0.0 on PyPi fails to install with python 2.7

### ISSUE THREAD

**Issue by kudangun** [chaimleib/intervaltree#77]
```
Collecting intervaltree (from pyOCD!=0.13.0->mbed-flasher==0.*,>=0.9.0->mbed-clitest->-r requirements.txt (line 1))
  Downloading https://files.pythonhosted.org/packages/7c/1c/20e098774a83f8eb8b9e90781d7570c00339f14acc412c91845f86842519/intervaltree-3.0.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-eq9tan/intervaltree/setup.py", line 29, in <module>
        from utils import version
    ImportError: No module named utils
```

**Comment by 0xc0170** [comment]
Came here to Github issues to check if this was already reported as our Travis started failing recently (latest version pushed 5h ago, about the same time as our CI started failing).

**Comment by chaimleib** [comment]
I can't reproduce using `pip uninstall intervaltree; pip install intervaltree==3.0.0`. I'll try on a different computer.

This version was tested using `make test` on my local, and with Travis. It was released with `~/Library/Python/3.7/bin/twine upload dist/*`.

**Comment by 0xc0170** [comment]
`pin install -U intervaltree` using python 2.7 fails.

```
pip 18.1

pip install -U intervaltree
Collecting intervaltree
  Using cached https://files.pythonhosted.org/packages/7c/1c/20e098774a83f8eb8b9e90781d7570c00339f14acc412c91845f86842519/intervaltree-3.0.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "c:\users\markoj01\appdata\local\temp\pip-install-2cuwvg\intervaltree\setup.py", line 29, in <module>
        from utils import version
    ImportError: No module named utils
```

Same was reproduced in our Travis

**Comment by 0xc0170** [comment]
Adding our Travis log here as well: https://travis-ci.org/ARMmbed/mbed-os/jobs/469068292

**Comment by 0xc0170** [comment]
Python 3.7 works , tested in the env. The latest version broke python 2.7 support. 

I checked travis config here it should be running tests with python 2.7 , not sure how it slipped.

**Comment by chaimleib** [comment]
It looks like the issue is that setup.py tries to import the utils. That is strange, since that import used to work in 2.1.0.

**Comment by chaimleib** [comment]
I'll try a release on testpypi, running `make distclean` first. That seems to solve certain errors, maybe it will solve this one too.

**Comment by RomeoActual** [comment]
2.1.0 setup.py doesn't seem to contain import utils.

import os
import errno
import sys
import subprocess
from warnings import warn
from setuptools import setup
from setuptools.command.test import test as TestCommand

import re

**Comment by flit** [comment]
The problem seems to be that neither `MANIFEST.in` or `setup.py` include the `utils` package in a source distribution. So when `setup.py` is run when installing via pip, the `utils` package is truly missing.

If you add `include utils/*` to `MANIFEST.in` the problem should be fixed.

**Comment by kaidokert** [comment]
Broke our stuff as well, on clean python 2.7.14 virtualenv same `from utils import version` exception.

**Comment by chaimleib** [comment]
I've released 3.0.1. I hope this fixes it!

**Comment by flit** [comment]
Looks good to me! 3.0.1 installs successfully in both Python 2 and 3. I think you can close this issue now. Thanks!
